app/e2e/tests/test-teardown.ts

```diff
@@ -0,0 +1,66 @@
+import { chromium } from '@playwright/test';
+import type { Page } from '@playwright/test';
+
+async function login(page: Page) {
+    await page.locator('input[type="email"]').fill('test@prytaneum.io');
+    await page.locator('input[type="password"]').fill('password');
+    await page.locator('text=LoginOr, register an account >> button').click();
+    await page.waitForURL('http://localhost:8080/organizations/me');
+}
```

Comment:
I think putting this inside a folder named `common/` would work. Then inside of `common/`, we'd have a `setup/` and `interactions/` folder. Setup would contain things that help setup data in the database maybe? Then `interactions` would have common user interactions like `loginWithPassword`.

Reply:
Further expanding on this I think it'd be pretty cool to have tests like so
```ts
import * as I from '@local/common/interactions';
import * as ThereIs from '@local/common/setup';

// ... in some test

// setup
ThereIs.aUserWithEmailAndPassword("email@example.com", "123");

I.navigateTo("/login");
I.loginWithEmailAndPassword("email@example.com", "123");

// insert assertions here
```
I think an API similar to this would be pretty cool / easy to reason about. I'm not sure how hard it is to setup, but I'm just throwing it out there.